File: C++/MathematicalPrograms/2DShapes/AreaAndPerimeterOfRhombus.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
	int diagonal1, diagonal2;
	int side;
	double area, perimeter;
	cout << "Enter the length of Diagonal 1";
	cin >> diagonal1;
	cout << "Enter the length of Diagonal 2";
	cin >> diagonal2;
	area = 0.5*diagonal1*diagonal2;
	cout << "Area of Rhombus is " << area << " sq.cm.";
	cout << "\nEnter the side of the Rhombus";
	cin >> side;
	perimeter = 4*side;
	cout << "Perimeter of the Rhombus is " << perimeter << " cm";
}

File: Java/MathematicalPrograms/2DShapes/AreaAndPerimeterOfRhombus.java
import java.util.Scanner;
public class AreaAndPerimeterOfRhombus {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter the length of Diagonal 1");
		int diagonal1 = myObj.nextInt();
		System.out.println("Enter the length of Diagonal 2");
		int diagonal2 = myObj.nextInt();
		double area = 0.5*diagonal1*diagonal2;
		System.out.println("Area of Rhombus is " + area + " sq.cm.");
		System.out.println("Enter the side of the Rhombus");
		int side = myObj.nextInt();
		double peri = 4*side;
		System.out.println("Perimeter of the Rhombus is " + peri);
	}
}